solutions/pytorch_tests/Makefile

```diff
@@ -0,0 +1,135 @@
+TOP = $(abspath ../..)
+include $(TOP)/defs.mak
+
+APPBUILDER=$(TOP)/scripts/appbuilder
+
+DOCKER_REGISTRY           = docker.io
+DOCKER_ORG                = mystikos
+DOCKER_IMAGE              = pytorch-test
+DOCKER_FULL_NAME          = $(DOCKER_REGISTRY)/$(DOCKER_ORG)/$(DOCKER_IMAGE)
+DOCKER_FILE               = Dockerfile
+DOCKER_TAG                = $(PYTORCH_VERSION)
+
+BASE_IMAGE                = python:3.9-buster
+PYTORCH_VERSION           = v1.10.0
+BUILD_PROGRESS            = auto
+BUILD_TARGET              = test
+BUILD_ARGS                = --build-arg BASE_IMAGE=$(BASE_IMAGE) \
+							--build-arg PYTORCH_VERSION=$(PYTORCH_VERSION)
+EXTRA_OPTIONS             = --progress=$(BUILD_PROGRESS) \
+							--target $(BUILD_TARGET) \
+							-t $(DOCKER_FULL_NAME):$(DOCKER_TAG) \
+							$(BUILD_ARGS)
+
+DOCKER_BUILD              = DOCKER_BUILDKIT=1 \
+							docker build \
+							-f $(DOCKER_FILE) \
+							$(EXTRA_ARGS) .
+DOCKER_PUSH               = docker push $(DOCKER_FULL_NAME):$(DOCKER_TAG)
+DOCKER_RUN                = docker run --rm -it $(DOCKER_FULL_NAME):$(DOCKER_TAG)
+
+RUN_PYTEST                = $(MYST_EXEC) rootfs $(OPTS) --app-config-path config.json /usr/local/bin/python -m pytest -v
+
+ifdef STRACE
+OPTS += --strace
+endif
+
+.PHONY: all
+all: rootfs
+
+.PHONY: image
+image:
+	$(DOCKER_BUILD)
+
+.PHONY: docker-run
+docker-run:
+	$(DOCKER_RUN) /usr/local/bin/pytest /workspace/pytorch/test/test_autograd.py -v
+
+.PHONY: push
+push:
+	$(DOCKER_PUSH)
```

Comment:
Just put the command here. We don't need so many defined variables

Reply:
Fixed.

Reply:
Fixed.